requirements.txt

```diff
@@ -3,11 +3,49 @@
 # This source code is licensed under the MIT license found in the
```

Comment:
This file should be update manually to make sur we always have to minor updates or patch. Look at the current packages you added here https://pypi.org/, and put a range starting from the current version to all the minor change 

Exemple
pytest>=5.4.3,<5.5.0

Reply:
not sure I understand this comment.

Reply:
do you want me to check each dependency and ensure that we are using the most recent version?